diff --git a/Makefile.am b/Makefile.am
index d86ce7e..5e25028 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -61,10 +61,6 @@ check_PROGRAMS = \
 
 EXTRA_DIST += \
        test/README \
-       test/bind_range \
-       test/checkaffinity \
-       test/checktopology \
-       test/numademo \
        test/printcpu \
        test/regress \
        test/regress2 \
@@ -131,13 +127,7 @@ test: all $(check_PROGRAMS) regress1 regress2 test_numademo
 TESTS_ENVIRONMENT = builddir='$(builddir)'; export builddir;
 
 TESTS = \
-       test/bind_range \
-       test/checkaffinity \
-       test/checktopology \
-       test/distance \
        test/move_pages \
-       test/nodemap \
-       test/numademo \
        test/regress \
        test/tbitmap
        
